planning/behavior_velocity_planner/src/scene_module/run_out/dynamic_obstacle.cpp

```diff
@@ -252,7 +252,8 @@ pcl::PointCloud<pcl::PointXYZ> transformPointCloud(
   pcl::PointCloud<pcl::PointXYZ> pointcloud_pcl;
   pcl::fromROSMsg(input_pointcloud, pointcloud_pcl);
   pcl::PointCloud<pcl::PointXYZ> pointcloud_pcl_transformed;
-  pcl::transformPointCloud(pointcloud_pcl, pointcloud_pcl_transformed, transform_matrix);
+  tier4_autoware_utils::transformPointCloud(
+    pointcloud_pcl, pointcloud_pcl_transformed, transform_matrix);
```

Comment:
The role of this `transformPointCloud` function defined in `dynamic_obstacle.cpp` seems to be duplicated with your function, so may be better to replace this whole function with `tier4_autoware_utils::transformPointCloud` (not necessarily in this PR though).

@takayuki5168 Can I ask your thought?

Reply:
This function: https://github.com/kaancolak/autoware.universe/blob/ab10e9c7f695b0bf6c39ece9e123798f6e1a0bf3/planning/behavior_velocity_planner/src/scene_module/run_out/dynamic_obstacle.cpp#L245-L259

Reply:
I agree with the Minoda-san's opinion.

Reply:
Minoda-san @kminoda, I am planning to create another PR about wrapping "pcl_ros::transformPointCloud" for handling zero points PointCloud2 messages with zero points.